insitu/models.py

```diff
@@ -190,6 +190,18 @@ def check_other_user(self, *args, **kwargs):
             self.requesting_user not in self.created_by.team.teammates.all()
         )
 
+    @transition_check('validate',)
+    def check_validator_user(self, *args, **kwargs):
+        """
+        Check if the user is the creator
```

Comment:
I think you should remove to lines below if you don't provide a description for those arguments.